src/hooks/useFilterBar.stories.mdx

```diff
@@ -0,0 +1,271 @@
+  const { onChange, onMixedStateChange, onClear, onClearAll, selectedItems } =
+    useFilterBar();
+
+  return (
+    <FilterBar
+      id="filterbar"
+      headingText="FilterBar"
+      layout="row"
+      selectedItems={selectedItems}
+      showClearAll={true}
+      onClear={onClearAll}
+      showSubmitAll={true}
+      onSubmit={() => {}}
+    >
+      <MultiSelectGroup
+        id="multiselect-group"
+        key="multiselect-group-key"
+        labelText="MultiSelect Group"
+      >
+        {multiSelectItems &&
+          multiSelectItems.map((multiSelect) => (
+            <MultiSelect
+              key={multiSelect.id}
+              id={multiSelect.id}
+              label={multiSelect.name}
+              variant="dialog"
+              items={multiSelect.items}
+              selectedItems={selectedItems}
+              onChange={(e) => {
+                onChange(e.target.id, multiSelect.id);
+              }}
+              onMixedStateChange={(e) => {
+                onMixedStateChange(
+                  e.target.id,
+                  multiSelect.id,
+                  multiSelect.items
+                );
+              }}
+              onClear={() => {
+                onClear(multiSelect.id);
+              }}
+              onApply={() => {}}
+            />
+          ))}
+      </MultiSelectGroup>
+    </FilterBar>
+  );
+}
+```
+
+## useFilterBar Hook implementation using setSelectedItems for a initial selection of filters
+
+```jsx
+const multiSelectItems = [
+  {
+    id: "colors",
+    name: "Colors",
+    items: [
+      { id: "red", name: "Red" },
+      { id: "blue", name: "Blue" },
+      { id: "yellow", name: "Yellow" },
+    ],
+  },
+  {
+    id: "pets",
+    name: "Pets",
+    items: [
+      { id: "cat", name: "Cat" },
+      {
+        id: "dog",
+        name: "Dog",
+        children: [
+          { id: "corgy", name: "Corgy" },
+          { id: "german-sheperd", name: "German Sheperd" },
+          { id: "afghan-hound", name: "Afghan Hound" },
+        ],
+      },
+      { id: "rat", name: "Rat" },
+    ],
+  },
+];
+```
+
+The required data structure to set the selectedItems object:
+
+```jsx
+// The selectedItems structur for one MultiSelect
+const initialFilter = {
+  colors: {
+    items: ["red"],
+  },
+};
+
+// The selectedItems structur for setting state on multiple filters
+const initialFilters = {
+  colors: {
+    items: ["red"],
+  },
+  locations: {
+    items: ["corgy", "german-sheperd"],
+  },
+};
+```
+
+The implemntation of the `useFilterBar` hook setting a inital selection of selectedItems in the MultiSelect.
+
+```jsx
+import React from "react";
+import {
+  FilterBar,
+  MultiSelectGroup,
+  MultiSelect,
+  useFilterBar,
+} from "@nypl/design-system-react-components";
+
+export default function FilterBarWithHookExample() {
+  const { onChange, onClear, onClearAll, setSelectedItems, selectedItems } =
+    useFilterBar();
+  setSelectedItems(initialFilter);
```

Comment:
This is nice but just define an example value for `initialFilter`.

Reply:
Hey @EdwinGuzman just to clarify: You suggest to remove the detailed example and just leave the data structure example code (line 199 to line 208) for the `initialFilter`? Just want to make sure I understand correctly.

Reply:
Done.